opencog/nlp/relex2logic/det.scm

```diff
@@ -0,0 +1,59 @@
+
+(define det
+    (BindLink
+        (VariableList
+            (TypedVariableLink
+                (VariableNode "$a-parse")
+                (TypeNode "ParseNode")
+            )
+            (TypedVariableLink
+                (VariableNode "$noun")
+                (TypeNode "WordInstanceNode")
+            )
+            (TypedVariableLink
+                (VariableNode "$det")
+                (TypeNode "WordInstanceNode")
+            )
+        )
+        (AndLink
+            (WordInstanceLink
+                (VariableNode "$noun")
+                (VariableNode "$a-parse")
+            )
+            (WordInstanceLink
+                (VariableNode "$det")
+                (VariableNode "$a-parse")
+            )
+            (EvaluationLink
+                (DefinedLinguisticRelationshipNode "_det")
+                (ListLink
+                    (VariableNode "$noun")
+                    (VariableNode "$det")
+                )
+            )
+            (InheritanceLink 
+                (VariableNode "$noun")
```

Comment:
I think the latest FC would apply all the R2L rules once only, so I guess there will be no 2nd step for R2L rules? Anyway this rule looks fine to me as well, relex can generate these atoms